### DISCUSSION TITLE
Switch between last used tabs (Ctrl+Tab)

### DISCUSSION THREAD

**Discussion by neocollin** [Alex313031/thorium#618] [Ideas]
I believe many people here comes from Brave. Thoughts about bringing Ctrl+Tab feature to be built-in to the browser itself?
Shortcuts always works better when one has 100+tabs open - LOL. I would use an extn, but being built in is always better!

**Comment by mikerowe81** [comment]
I would also love to see this feature brought back.

**Comment by BenLangers** [comment]
+1(00000000)

**Comment by MAX-786** [comment]
is it there yet? otherwise +∞

**Comment by gz83** [comment]
Version 150 already includes this feature.
